v2/googlecloud-to-splunk/docs/GCSToSplunk/README.md

```diff
@@ -0,0 +1,237 @@
+# GCS To Splunk Dataflow Template
+
+The [GCSToSplunk](../../src/main/java/com/google/cloud/teleport/v2/templates/GCSToSplunk.java) pipeline ingests
+data from CSV files stored in GCS, optionally executes a UDF, converts the output to [SplunkEvent](https://github.com/apache/beam/blob/master/sdks/java/io/splunk/src/main/java/org/apache/beam/sdk/io/splunk/SplunkEvent.java) objects,
```

Comment:
I don't think we need to describe converting to a `SplunkEvent`. This is more describing what it does, not how it does it, and the specific objects we use is more "how" rather than "what".